plugins/org.jkiss.dbeaver.ui.editors.sql.terminal/plugin.xml

```diff
@@ -29,7 +29,12 @@
 
     <extension point="org.eclipse.ui.handlers">
         <handler commandId="org.jkiss.dbeaver.ui.editors.sql.show.terminalView" class="org.jkiss.dbeaver.ui.editors.sql.terminal.SQLTerminalViewSwitchHandler">
-            <enabledWhen><reference definitionId="org.jkiss.dbeaver.core.ui.sql.editor"/></enabledWhen>
+            <enabledWhen>
```

Comment:
Why we disable teminal view? It is just a plaintextr presentation and it makes sense for managers.